packages/frontend/src/locales/en.json

```
@@ -35,9 +55,10 @@
     },
     "discordSocket": {
       "channelLabel": "Discord channel",
-      "channelPlaceholder": "New channel-ID",
+      "channelPlaceholder": "1234567890123456",
+      "description": "Please enter the required values into the form. If problem occurs or you do not know what to do - You can find a manual <a href=\"{0}\">here</a>",
```

Sounds a bit strange. I would suggest either
```suggestion
      "description": "Please enter the required values into the form. If a problem occurs or you do not know what to do - You can find a manual <a href=\"{0}\">here</a>",
```

or

```suggestion
      "description": "Please enter the required values into the form. If problems occur or you do not know what to do - You can find a manual <a href=\"{0}\">here</a>",
```